webapps-schema/src/main/java/io/camunda/webapps/schema/entities/AbstractExporterEntity.java

```diff
@@ -5,42 +5,45 @@
  * Licensed under the Camunda License 1.0. You may not use this file
  * except in compliance with the Camunda License 1.0.
  */
-package io.camunda.operate.entities;
+package io.camunda.webapps.schema.entities;
 
-public abstract class OperateEntity<T extends OperateEntity<T>> {
+import java.util.Objects;
+
+/**
+ * Represents an entity that can be written to ElasticSearch or OpenSearch
+ *
+ * @param <T>
+ */
+public abstract class AbstractExporterEntity<T extends AbstractExporterEntity<T>>
```

Comment:
:question: Can this be merged with `OperateZeebeEntity`?

Reply:
@deepthidevaki I decided not to merge them, so that other web apps can reuse `AbstractExporterEntity`. `OperateZeebeEntity` contains two more fields additionally that other web apps may not need.